### DISCUSSION TITLE
[Art competition] Public-Key Pair

### DISCUSSION THREAD

**Discussion by adzaky15** [prosyslab-classroom/cs348-information-security#408] [Art competition]
![DALL·E 2023-05-24 20 49 23 - painting of heaven (right) and hell (left)](https://github.com/prosyslab-classroom/cs348-information-security/assets/89621454/c167c8f4-0310-4306-aec8-1104ce2b9520)

## Short Description
For me, "public key-pair" means "painting of heaven (right) and hell (left)".

## Long Description (up to 3 sentences)
Public key is the shortage of water (left), while private key is the abundance of water (right). These keys are similar, yet different. This painting is purposefully asymmetric.

**Comment by KihongHeo** [comment]
Interesting. Why do you think that "Public key is the shortage of water (left), while private key is the abundance of water (right)"?
